packages/server/src/api/controllers/query/index.ts

```diff
@@ -214,11 +219,11 @@ async function execute(ctx: any, opts = { rowsOnly: false }) {
 }
 
 export async function executeV1(ctx: any) {
-  return execute(ctx, { rowsOnly: true })
+  return execute(ctx, { rowsOnly: true, isAutomation: false })
 }
 
-export async function executeV2(ctx: any) {
-  return execute(ctx, { rowsOnly: false })
+export async function executeV2(ctx: any, isAutomation?: any) {
```

Comment:
Preferably all boolean parameters should be wrapped in objects, options objects like `ctx: any, { isAutomation }: { isAutomation?: boolean } = {}`

Avoids situations with calls like `executeV2(ctx, false)` - difficult to see what this does vs `executeV2(ctx, { isAutomation: false })`